in-cluster/default-kurl.yaml

```diff
@@ -4,6 +4,14 @@ metadata:
   name: default
 spec:
   collectors:
+    # Note: apiserver logs are owned by root so for this collector
+    # to succeed it requires sudo privileges for the user
+    - copyFromHost:
+        collectorName: "copy apiserver audit logs"
+        image: busybox:1
+        hostPath: "/var/log/apiserver/"
+        name: "logs"
+        extractArchive: true
     - copyFromHost:
         collectorName: "copy kURL logs"
         image: busybox:1
```

Comment:
there's another copyFromHost collector below, `kurl-host-preflights` - can we make sure to use the same image as it?

(I don't care which one changes)

Reply:
I went with `alpine`.